bgpd/bgp_vty.c

```diff
@@ -7387,6 +7387,35 @@ bgp_show_peer_afi (struct vty *vty, struct peer *p, afi_t afi, safi_t safi,
     }
 }
 
+static const char *
+text_sanitize (char *buf, size_t bufsz, const char *in, size_t inlen)
```

Comment:
From p->notify.length.  I feel like there is some subtlety to the question that I am missing.